### ISSUE TITLE
Reliability higher than one in ordinal CFA and theta parameterization

### ISSUE THREAD

**Issue by hynekcigler** [simsem/semTools#82]
Hi, I got an error using `reliability` function over the ordinal CFA model with theta parameterization. Even in simple unidimensional model, I got reliability higher than one. 

Data are [available here](https://www.dropbox.com/s/22as08tpsevnv50/GAD_data.rdata?dl=0), here is the model definition:
`model_1 <- "G =~ GAD7_1 + GAD7_2 + GAD7_3 + GAD7_4 + GAD7_5 + GAD7_6 + GAD7_7"`

Fitting continuous CFA model leads to plausible reliability estimate:
```
cfa1 <- cfa(model = model_1, data = data, ordered = FALSE, orthogonal = T, 
estimator = "MLR", missing = "fiml", std.lv = TRUE)
reliability(cfa1)
               G
alpha  0.8829656
omega  0.8862292
omega2 0.8862292
omega3 0.8855434
avevar 0.5324595
```
the same with ordinal CFA with delta parameterization:
```
cfa2 <- cfa(model = model_1, data = data, ordered = TRUE, orthogonal = T, 
estimator = "WLSMV", missing = "pairwise", parameterization = "delta", std.lv = TRUE)
reliability(cfa2)
               G
alpha  0.9163732
omega  0.8857005
omega2 0.8857005
omega3 0.8885252
avevar 0.6177499
```

Unfortunately, the same model with theta parameterization gives an obvious error:
```
cfa3 <- cfa(model = model_1, data = data, ordered = TRUE, orthogonal = T, 
estimator = "WLSMV", missing = "pairwise", parameterization = "theta", std.lv = TRUE)
reliability(cfa3)
               G
alpha  0.9163732
omega  1.5499746
omega2 1.5499746
omega3 1.5549181
avevar 0.6387913
```

I assume this is an error caused by wrong accounting for thresholds as Green and Young (2001, formula 21) suggested.

**Comment by TDJorgensen** [comment]
Thanks for making this a formal issue.  Someone emailed me about this issue just a few weeks ago, and I couldn't see anything wrong in the hidden `omegaCat()` function because it standardized thresholds before calculating cumulative probabilities anyway.  But now I see the actual problem is in the hidden `getThreshold()` function, which was grabbing the model-implied *standardized* thresholds rather than the estimated model parameters themselves.  The former were interpreted as being on the latent-response scale implied by the reciprocal of the scaling factors, rather than as standardized.  Of course, under the default delta parameterization (without additional constraints), thresholds are already standardized because scaling factors are fixed to 1; but that is not the case with the theta parameterization.

A similar mistake was in only passing the model-implied polychoric correlations to the hidden `omegaCat()` function, rather than the model-implied covariance matrix on the latent-response scale (again, which is not standardized under theta parameterization).

Both problems fixed now in the development version (0.5-3.911):

```
devtools::install_github("simsem/semTools/semTools")
```

Running your syntax reproduces your delta-method reliability estimates above, which theta-parameterization now matches to the 6th decimal place.

**Comment by hynekcigler** [comment]
Thank you very much for a quick fix! :-)
